build.sh

```diff
@@ -37,26 +38,26 @@ INSTALL_PREFIX=""
 CMAKE_OPTS=""
 BUILD_TARGET="${MACHINE_ID:-'localhost'}"
 BUILD_VERBOSE="NO"
-ADD_RRFS_TESTS="YES"
 CLEAN_BUILD="NO"
 BUILD_JCSDA="YES"
+BUILD_SUPER_EXE="NO"
 DYCORE="FV3andMPAS"
 COMPILER="${COMPILER:-intel}"
 
-while getopts "p:t:c:m:hvfs-:" opt; do
+while getopts "p:c:m:j:hvfsx-:" opt; do
```

Comment:
@SamuelDegelia-NOAA Sorry I forgot to remove `-`  when I removed the `--notestdata` part. Could you remove it here? or I can create a separate PR to do this. It is up to you. Thanks!

Reply:
Just updated it.

Reply:
Thank you, @SamuelDegelia-NOAA